src/types/index.ts

```diff
@@ -167,6 +167,12 @@ export interface Tenants {
     gcm_account_id?: string | null;
 }
 
+export interface TenantHidesDataPreview {
```

Comment:
Did you intentionally make _hides_ plural in the name of this interface?

Reply:
The s is on purpose it is not a plural.

Reply:
I would have said _why did you use the present simple tense of the verb **hide**?_ but most people don't know the difference between the present tense and the present simple tense. Haha so I use _plural_ loosely (and incorrectly) to indicate a word ending in an S. The average English speaker understands what I am communicating and never detects the inaccuracy, so I have rolled with it over the years. I'll be very specific moving forward.